cpp/benchmarks/join/join_common.hpp

```diff
@@ -35,6 +40,18 @@
 
 #include "generate_input_tables.cuh"
 
+struct null75_generator {
```

Comment:
Would something like this be useful in other benchmarks? Should we make it parametrizable and move it to some common file?

Reply:
Yes. I have a parametrizable code for other benchmarks in a common file. That will be in a different PR, together with more changes.